### PR TITLE
lmp/jobserv: add Code Review base targets for imx6ul*

### PR DESCRIPTION
**Pull request by MrCry0**
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>

### CODE REVIEW

**Review by doanac — COMMENTED** [review]
i'll defer to ricardo unless he's not around and you just need this quick. if that's the case feel free to merge.

**Comment by MrCry0** [comment]
> i'll defer to ricardo unless he's not around and you just need this quick. if that's the case feel free to merge.

@doanac these base images are needed to test releases; this fix allows us to avoid building these images locally and just use reference ones. I can't merge it without +1 :( If you don't mind, add your one.

**Review by doanac — APPROVED** [review]

**Comment by MrCry0** [comment]
Ah, I even can't merge it with approvals :) 
`The base branch restricts merging to authorized users`

**Pull request merged**
